2015-12-18 23:10:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE IF NOT EXISTS ([*],  CLOB, ,  OTHER, ,  DATE, PRIMARY KEY ()"; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS ([*],  CLOB, ,  OTHER, ,  DATE, PRIMARY KEY ()"; expected "identifier"; SQL statement:
CREATE TABLE IF NOT EXISTS (,  CLOB, ,  OTHER, ,  DATE, PRIMARY KEY () [42001-190]
2015-12-18 23:12:55 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]" [50000-190]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:373)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:356)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2888)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2450)
	at org.h2.engine.Database.open(Database.java:672)
	at org.h2.engine.Database.openDatabase(Database.java:269)
	at org.h2.engine.Database.<init>(Database.java:263)
	at org.h2.engine.Engine.openSession(Engine.java:65)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:190)
	at org.h2.jdbcx.JdbcDataSource.getConnection(JdbcDataSource.java:161)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseService.<init>(DatabaseService.java:34)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseServiceTest.setUp(DatabaseServiceTest.java:17)
	at junit.framework.TestCase.runBare(TestCase.java:139)
	at junit.framework.TestResult$1.protect(TestResult.java:122)
	at junit.framework.TestResult.runProtected(TestResult.java:142)
	at junit.framework.TestResult.run(TestResult.java:125)
	at junit.framework.TestCase.run(TestCase.java:129)
	at junit.framework.TestSuite.runTest(TestSuite.java:255)
	at junit.framework.TestSuite.run(TestSuite.java:250)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:84)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]" [50000-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 42 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:342)
	... 37 more
2015-12-18 23:13:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE IF NOT EXISTS ([*],  CLOB, ,  OTHER, ,  DATE, PRIMARY KEY ()"; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS ([*],  CLOB, ,  OTHER, ,  DATE, PRIMARY KEY ()"; expected "identifier"; SQL statement:
CREATE TABLE IF NOT EXISTS (,  CLOB, ,  OTHER, ,  DATE, PRIMARY KEY () [42001-190]
2015-12-18 23:13:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE IF NOT EXISTS ([*] CLOB,  INT,  DATE, PRIMARY KEY ()"; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS ([*] CLOB,  INT,  DATE, PRIMARY KEY ()"; expected "identifier"; SQL statement:
CREATE TABLE IF NOT EXISTS ( CLOB,  INT,  DATE, PRIMARY KEY () [42001-190]
2015-12-18 23:16:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE IF NOT EXISTS ([*] CLOB,  INT,  DATE, PRIMARY KEY ()"; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS ([*] CLOB,  INT,  DATE, PRIMARY KEY ()"; expected "identifier"; SQL statement:
CREATE TABLE IF NOT EXISTS ( CLOB,  INT,  DATE, PRIMARY KEY () [42001-190]
2015-12-18 23:16:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: ","
Unknown data type: ","; SQL statement:
CREATE TABLE IF NOT EXISTS student_of_mipt( CLOB,  INT,  DATE, PRIMARY KEY () [50004-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4009)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:3872)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5808)
	at org.h2.command.Parser.parseCreate(Parser.java:4167)
	at org.h2.command.Parser.parsePrepared(Parser.java:349)
	at org.h2.command.Parser.parse(Parser.java:304)
	at org.h2.command.Parser.parse(Parser.java:276)
	at org.h2.command.Parser.prepareCommand(Parser.java:241)
	at org.h2.engine.Session.prepareLocal(Session.java:460)
	at org.h2.engine.Session.prepareCommand(Session.java:402)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseService.createTable(DatabaseService.java:140)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseServiceTest.testCreateTable(DatabaseServiceTest.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at junit.framework.TestCase.runTest(TestCase.java:176)
	at junit.framework.TestCase.runBare(TestCase.java:141)
	at junit.framework.TestResult$1.protect(TestResult.java:122)
	at junit.framework.TestResult.runProtected(TestResult.java:142)
	at junit.framework.TestResult.run(TestResult.java:125)
	at junit.framework.TestCase.run(TestCase.java:129)
	at junit.framework.TestSuite.runTest(TestSuite.java:255)
	at junit.framework.TestSuite.run(TestSuite.java:250)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:84)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-12-18 23:22:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: ","
Unknown data type: ","; SQL statement:
CREATE TABLE IF NOT EXISTS student_of_mipt( CLOB,  INT, birthday DATE, PRIMARY KEY () [50004-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4009)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:3872)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5808)
	at org.h2.command.Parser.parseCreate(Parser.java:4167)
	at org.h2.command.Parser.parsePrepared(Parser.java:349)
	at org.h2.command.Parser.parse(Parser.java:304)
	at org.h2.command.Parser.parse(Parser.java:276)
	at org.h2.command.Parser.prepareCommand(Parser.java:241)
	at org.h2.engine.Session.prepareLocal(Session.java:460)
	at org.h2.engine.Session.prepareCommand(Session.java:402)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseService.createTable(DatabaseService.java:144)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseServiceTest.testCreateTable(DatabaseServiceTest.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at junit.framework.TestCase.runTest(TestCase.java:176)
	at junit.framework.TestCase.runBare(TestCase.java:141)
	at junit.framework.TestResult$1.protect(TestResult.java:122)
	at junit.framework.TestResult.runProtected(TestResult.java:142)
	at junit.framework.TestResult.run(TestResult.java:125)
	at junit.framework.TestCase.run(TestCase.java:129)
	at junit.framework.TestSuite.runTest(TestSuite.java:255)
	at junit.framework.TestSuite.run(TestSuite.java:250)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:84)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-12-18 23:28:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE IF NOT EXISTS STUDENT_OF_MIPT(NAME CLOB, GROUP[*] INT, BIRTHDAY DATE, PRIMARY KEY (NAME) "; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS STUDENT_OF_MIPT(NAME CLOB, GROUP[*] INT, BIRTHDAY DATE, PRIMARY KEY (NAME) "; expected "identifier"; SQL statement:
CREATE TABLE IF NOT EXISTS student_of_mipt(name CLOB, group INT, birthday DATE, PRIMARY KEY (name) [42001-190]
2015-12-18 23:35:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE IF NOT EXISTS RATING_OF_STUDENTS(NAME CLOB, RANK INT, BIRTHDAY DATE, PRIMARY KEY (NAME) [*]"; ожидалось "INDEX, ,, )"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS RATING_OF_STUDENTS(NAME CLOB, RANK INT, BIRTHDAY DATE, PRIMARY KEY (NAME) [*]"; expected "INDEX, ,, )"; SQL statement:
CREATE TABLE IF NOT EXISTS rating_of_students(name CLOB, rank INT, birthday DATE, PRIMARY KEY (name) [42001-190]
2015-12-18 23:43:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "CLOBPRIMARY"
Unknown data type: "CLOBPRIMARY"; SQL statement:
CREATE TABLE IF NOT EXISTS rating_of_students(name CLOBPRIMARY KEY, rank INT, birthday DATE,  [50004-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4009)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:3872)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5808)
	at org.h2.command.Parser.parseCreate(Parser.java:4167)
	at org.h2.command.Parser.parsePrepared(Parser.java:349)
	at org.h2.command.Parser.parse(Parser.java:304)
	at org.h2.command.Parser.parse(Parser.java:276)
	at org.h2.command.Parser.prepareCommand(Parser.java:241)
	at org.h2.engine.Session.prepareLocal(Session.java:460)
	at org.h2.engine.Session.prepareCommand(Session.java:402)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseService.createTable(DatabaseService.java:148)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseServiceTest.testCreateTable(DatabaseServiceTest.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at junit.framework.TestCase.runTest(TestCase.java:176)
	at junit.framework.TestCase.runBare(TestCase.java:141)
	at junit.framework.TestResult$1.protect(TestResult.java:122)
	at junit.framework.TestResult.runProtected(TestResult.java:142)
	at junit.framework.TestResult.run(TestResult.java:125)
	at junit.framework.TestCase.run(TestCase.java:129)
	at junit.framework.TestSuite.runTest(TestSuite.java:255)
	at junit.framework.TestSuite.run(TestSuite.java:250)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:84)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-12-18 23:44:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE IF NOT EXISTS RATING_OF_STUDENTS(NAME CLOB PRIMARY KEY, RANK INT, BIRTHDAY DATE,  [*]"; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS RATING_OF_STUDENTS(NAME CLOB PRIMARY KEY, RANK INT, BIRTHDAY DATE,  [*]"; expected "identifier"; SQL statement:
CREATE TABLE IF NOT EXISTS rating_of_students(name CLOB PRIMARY KEY, rank INT, birthday DATE,  [42001-190]
2015-12-18 23:45:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE IF NOT EXISTS RATING_OF_STUDENTS(NAME CLOB PRIMARY KEY, RANK INT, BIRTHDAY DATE [*]"; ожидалось "(, FOR, UNSIGNED, NOT, NULL, AS, DEFAULT, GENERATED, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS RATING_OF_STUDENTS(NAME CLOB PRIMARY KEY, RANK INT, BIRTHDAY DATE [*]"; expected "(, FOR, UNSIGNED, NOT, NULL, AS, DEFAULT, GENERATED, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE IF NOT EXISTS rating_of_students(name CLOB PRIMARY KEY, rank INT, birthday DATE [42001-190]
2015-12-18 23:46:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Данная функция не поддерживается: "Index on BLOB or CLOB column: NAME CLOB NOT NULL"
Feature not supported: "Index on BLOB or CLOB column: NAME CLOB NOT NULL"; SQL statement:
CREATE TABLE IF NOT EXISTS rating_of_students(name CLOB PRIMARY KEY, rank INT, birthday DATE) [50100-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:216)
	at org.h2.index.BaseIndex.checkIndexColumnTypes(BaseIndex.java:78)
	at org.h2.mvstore.db.MVSecondaryIndex.<init>(MVSecondaryIndex.java:56)
	at org.h2.mvstore.db.MVTable.addIndex(MVTable.java:503)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:136)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:171)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:253)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseService.createTable(DatabaseService.java:151)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseServiceTest.testCreateTable(DatabaseServiceTest.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at junit.framework.TestCase.runTest(TestCase.java:176)
	at junit.framework.TestCase.runBare(TestCase.java:141)
	at junit.framework.TestResult$1.protect(TestResult.java:122)
	at junit.framework.TestResult.runProtected(TestResult.java:142)
	at junit.framework.TestResult.run(TestResult.java:125)
	at junit.framework.TestCase.run(TestCase.java:129)
	at junit.framework.TestSuite.runTest(TestSuite.java:255)
	at junit.framework.TestSuite.run(TestSuite.java:250)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:84)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-12-19 00:39:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Недопустимое значение "0" для параметра "parameterIndex"
Invalid value "0" for parameter "parameterIndex" [90008-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:228)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1382)
	at org.h2.jdbc.JdbcPreparedStatement.setObject(JdbcPreparedStatement.java:460)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseService.insert(DatabaseService.java:147)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseServiceTest.test(DatabaseServiceTest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-12-19 00:40:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Недопустимое значение "0" для параметра "parameterIndex"
Invalid value "0" for parameter "parameterIndex" [90008-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:228)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1382)
	at org.h2.jdbc.JdbcPreparedStatement.setObject(JdbcPreparedStatement.java:460)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseService.insert(DatabaseService.java:147)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseServiceTest.test(DatabaseServiceTest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
2015-12-19 00:41:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Недопустимое значение "0" для параметра "parameterIndex"
Invalid value "0" for parameter "parameterIndex" [90008-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:228)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1382)
	at org.h2.jdbc.JdbcPreparedStatement.setObject(JdbcPreparedStatement.java:460)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseService.insert(DatabaseService.java:147)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseServiceTest.test(DatabaseServiceTest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
2015-12-19 00:41:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Столбец "VARCHAR" не найден
Column "VARCHAR" not found; SQL statement:
SELECT * FROM rating_of_students WHERE VARCHAR = ? [42122-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.Comparison.optimize(Comparison.java:178)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:460)
	at org.h2.engine.Session.prepareCommand(Session.java:402)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:72)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:276)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseService.queryById(DatabaseService.java:90)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseServiceTest.test(DatabaseServiceTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-12-19 00:54:40 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]" [50000-190]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:373)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:356)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2888)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2450)
	at org.h2.engine.Database.open(Database.java:672)
	at org.h2.engine.Database.openDatabase(Database.java:269)
	at org.h2.engine.Database.<init>(Database.java:263)
	at org.h2.engine.Engine.openSession(Engine.java:65)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:190)
	at org.h2.jdbcx.JdbcDataSource.getConnection(JdbcDataSource.java:161)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseService.<init>(DatabaseService.java:34)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseServiceTest.test(DatabaseServiceTest.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]" [50000-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 51 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:342)
	... 46 more
2015-12-19 01:09:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Функция "VARCHAR" не найдена
Function "VARCHAR" not found; SQL statement:
SELECT * FROM rating_of_students WHERE VARCHAR(256) = ? [90022-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.readJavaFunction(Parser.java:2355)
	at org.h2.command.Parser.readFunction(Parser.java:2407)
	at org.h2.command.Parser.readTerm(Parser.java:2741)
	at org.h2.command.Parser.readFactor(Parser.java:2258)
	at org.h2.command.Parser.readSum(Parser.java:2245)
	at org.h2.command.Parser.readConcat(Parser.java:2215)
	at org.h2.command.Parser.readCondition(Parser.java:2065)
	at org.h2.command.Parser.readAnd(Parser.java:2037)
	at org.h2.command.Parser.readExpression(Parser.java:2029)
	at org.h2.command.Parser.parseSelectSimple(Parser.java:1986)
	at org.h2.command.Parser.parseSelectSub(Parser.java:1867)
	at org.h2.command.Parser.parseSelectUnion(Parser.java:1688)
	at org.h2.command.Parser.parseSelect(Parser.java:1676)
	at org.h2.command.Parser.parsePrepared(Parser.java:432)
	at org.h2.command.Parser.parse(Parser.java:304)
	at org.h2.command.Parser.parse(Parser.java:276)
	at org.h2.command.Parser.prepareCommand(Parser.java:241)
	at org.h2.engine.Session.prepareLocal(Session.java:460)
	at org.h2.engine.Session.prepareCommand(Session.java:402)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:72)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:276)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseService.queryById(DatabaseService.java:90)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseServiceTest.test(DatabaseServiceTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-12-19 01:12:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE IF NOT EXISTS NULL[*]( "; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS NULL[*]( "; expected "identifier"; SQL statement:
CREATE TABLE IF NOT EXISTS null( [42001-190]
2015-12-19 01:14:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Функция "VARCHAR" не найдена
Function "VARCHAR" not found; SQL statement:
SELECT * FROM rating_of_students WHERE VARCHAR(256) = ? [90022-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.readJavaFunction(Parser.java:2355)
	at org.h2.command.Parser.readFunction(Parser.java:2407)
	at org.h2.command.Parser.readTerm(Parser.java:2741)
	at org.h2.command.Parser.readFactor(Parser.java:2258)
	at org.h2.command.Parser.readSum(Parser.java:2245)
	at org.h2.command.Parser.readConcat(Parser.java:2215)
	at org.h2.command.Parser.readCondition(Parser.java:2065)
	at org.h2.command.Parser.readAnd(Parser.java:2037)
	at org.h2.command.Parser.readExpression(Parser.java:2029)
	at org.h2.command.Parser.parseSelectSimple(Parser.java:1986)
	at org.h2.command.Parser.parseSelectSub(Parser.java:1867)
	at org.h2.command.Parser.parseSelectUnion(Parser.java:1688)
	at org.h2.command.Parser.parseSelect(Parser.java:1676)
	at org.h2.command.Parser.parsePrepared(Parser.java:432)
	at org.h2.command.Parser.parse(Parser.java:304)
	at org.h2.command.Parser.parse(Parser.java:276)
	at org.h2.command.Parser.prepareCommand(Parser.java:241)
	at org.h2.engine.Session.prepareLocal(Session.java:460)
	at org.h2.engine.Session.prepareCommand(Session.java:402)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:72)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:276)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseService.queryById(DatabaseService.java:90)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseServiceTest.test(DatabaseServiceTest.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-12-19 01:24:40 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]" [50000-190]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:373)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:356)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2888)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2450)
	at org.h2.engine.Database.open(Database.java:672)
	at org.h2.engine.Database.openDatabase(Database.java:269)
	at org.h2.engine.Database.<init>(Database.java:263)
	at org.h2.engine.Engine.openSession(Engine.java:65)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:190)
	at org.h2.jdbcx.JdbcDataSource.getConnection(JdbcDataSource.java:161)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseService.<init>(DatabaseService.java:34)
	at ru.fizteh.fivt.students.nikitarykov.miniorm.DatabaseServiceTest.test(DatabaseServiceTest.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
Caused by: org.h2.jdbc.JdbcSQLException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]" [50000-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 51 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Development/Java/fizteh-java-2015/nikitarykov/test.mv.db [1.4.190/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:342)
	... 46 more
